### PR TITLE
The most reliable way to inspect html element

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

File: src/client/utils/dom.js [hunk 1]
```diff
@@ -404,55 +411,55 @@ export function isIframeWithoutSrc (iframe) {
 }
 
 export function isImgElement (el) {
-    return getTagName(el) === 'img' && isDomElement(el);
+    return instanceToString(el) === '[object HTMLImageElement]';
```
Comment (miherlosev) [inline src/client/utils/dom.js]: Please add pefomance comparison using jsperf.com

Comment (LavrovArtem) [inline src/client/utils/dom.js]: https://jsperf.com/props-vs-tostring01

Comment (miherlosev) [inline src/client/utils/dom.js]: Ok.

Comment (LavrovArtem) [inline src/client/utils/dom.js]: I don't think that it has any effects to performance.

Comment (miherlosev) [inline src/client/utils/dom.js]: Ok. I checked it and and it doesn't affect perfomance.

File: src/client/utils/dom.js [hunk 2]
```diff
@@ -647,15 +655,15 @@ export function isDocumentFragmentNode (el) {
...
 
 export function isTableElement (el) {
-    return getTagName(el) === 'table' && isDomElement(el);
+    return instanceToString(el) === '[object HTMLTableElement]';
 }
 
-export function isTableDataElement (el) {
-    return getTagName(el) === 'td' && isDomElement(el);
+export function isTableCellElement (el) {
+    return instanceToString(el) === NATIVE_TABLE_CELL_STR;
```
Comment (miherlosev) [inline src/client/utils/dom.js]: In standart it named as `TableDataCell`.
https://www.w3.org/TR/html401/struct/tables.html#h-11.2.6

`TableCell` is common name.
There are th - table header cell, td - table data cell.
So we should rename it to the `isTableDataCellElement`.

Comment (LavrovArtem) [inline src/client/utils/dom.js]: ok

**Pull request merged**